theforeman.org/scripts/debian/stage_packages.sh

```diff
@@ -7,19 +7,41 @@ set -xe
 # * the web node must have the freight class applied
 
 # Find the deps in the build-dir from the previous step
-DEB_PATH=./debian/${os}/build-${project}
+DEB_PATH=$(find ./ -name build-${project})
 
 # Upload all builds to stagingdeb for testing
-echo "scratch build: uploading to stagingdeb/${os}/${repoowner}-${version}"
+HOSTS="web01.osuosl.theforeman.org"
 export RSYNC_RSH="ssh -i /home/jenkins/workspace/staging_key/rsync_freightstage_key"
 USER=freightstage
-HOSTS="web01.osuosl"
-COMPONENT=${repoowner}-${version}
+
+if [[ "$DEB_PATH" == *"plugins"* ]];then
+  # Don't upload scratch builds to deb.tf.o
+  if [ x$repoowner = xtheforeman ] && [ x$pr_number = x ]; then
+    echo "Built from main repo, uploading to deb/plugins/main"
+    export RSYNC_RSH="ssh -i /home/jenkins/workspace/deb_key/rsync_freight_key"
```

Comment:
This is already set
```suggestion
```

Reply:
No, it's not. It's set to `/home/jenkins/workspace/staging_key/rsync_freightstage_key`, and that key can't write to `freight@`.

Reply:
Ah my bad. I'm addressing my caffeine levels right now.